### PR TITLE
write_redis plugin: Add max_set_duration to set duration for value

### PR DESCRIPTION
**Pull request by s1061123**
This change introduces max_set_duration, the duration for the value
as max_set_size. max_set_size can keep the db size, however, sometime
max_set_size allows to contain old data. max_set_duration can keep
the latest data.

### CODE REVIEW

File: src/write_redis.c [hunk 1]
```diff
@@ -125,6 +126,16 @@ static int wr_write(const data_set_t *ds, /* {{{ */
       freeReplyObject(rr);
   }
 
+  if (node->max_set_duration > 0) {
+    rr = redisCommand(node->conn, "ZREMRANGEBYSCORE %s -1 (%d", key,
```
Comment (octo) [inline src/write_redis.c]: The not-matched open parenthesis in `(%d` looks suspicious. If this in on purpose, please add a comment explaining it.

Comment (s1061123) [reply src/write_redis.c]: @octo, Thank you for your comment! '(%d' means 'less than %d' of redis score. I will update it soon.

Comment (s1061123) [reply src/write_redis.c]: @octo I've added comments in latest diff. Thanks again!

**Comment by s1061123** [comment]
Missing comment for documenting (src/collectd.conf.pod and src/collectd.conf.in), hence I will file another pull request for it.

**Pull request merged**